.vscode/launch.template.json

```diff
@@ -1,5 +1,17 @@
-// Rename this file 'launch.json' or merge its
-// contents into your existing configurations.
+/* 
+
+  Rename this file 'launch.json' or merge its
```

Comment:
I realized I should have said “copy this file to” instead of “rename this file,“ since renaming would show up as a deletion in git—want to update that while you’re at it?

Reply:
Aye